Core/include/EventTuple.h

```diff
@@ -122,6 +122,8 @@ using Point3D = analysis::Point3D_Float;
     VAR(Int_t, period) /* datataking period */ \
 	/* Event Weights Variables */ \
     VAR(Double_t, weight_pu) \
+    VAR(Double_t, weight_pu_up) \
+    VAR(Double_t, weight_pu_down) \
```

Comment:
- Weights stored in EventTuple should be reviewed. The ones that are computed directly in BaseAnalyzer shouldn't be stored.
- For example, pu weights (central/up/down) can be computed at the level of BaseAnalyzer, so shouldn't be stored in EventTuple
- On the other hand, in the summary tuple following 4 branches should be present and correctly filled: totalShapeWeight, totalShapeWeight_withTopPt, totalShapeWeight_withPuUp, totalShapeWeight_withPuDown.
- pu up/down should be implemented in [PileUpWeight](https://github.com/hh-italian-group/h-tautau/blob/master/McCorrections/src/PileUpWeight.cpp) and corresponding root files added for all 3 years.

Reply:
Please, cleanup unused weights. E.g. `weight_lepton_trig`, `weight_lepton_id_iso` etc.